sdk/tables/aztable/byte_array_response.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+// ByteArrayResponse converts the MapOfInterfaceResponse.Value from a map[string]interface{} to a []byte
+type ByteArrayResponse struct {
```

Comment:
Is this an interim step to do this as part of generated code, or do we plan to always do this conversion?

Reply:
We always plan to do this conversion. This change gives the customer the abiltity to use the `json.Unmarshal` functionality on recieved entities.